Source/FilePreviewPanel.cpp

```
@@ -20,21 +22,118 @@ FilePreviewPanel::FilePreviewPanel() {
+            setLoading(true);
+            std::string path = currentFile_->path;
+            loadedFilePath_ = path;
+            cancelLoading_ = false;
+            waveformReady_ = false;
+            
+            // Launch async task
+            loadFuture_ = std::async(std::launch::async, [this, path]() {
+                std::vector<float> rawAudio;
+                uint32_t rate = 0;
+                uint32_t channels = 0;
+                
```

Cppcheck (2.19.0)

[style] The function 'getCurrentValue' is never used.

(unusedFunction)